src/main/java/com/exactpro/th2/fix/client/Main.java

```diff
@@ -132,8 +138,15 @@ public static void main(String[] args) throws Exception {
 
         for (FixBean fixBean : settings.sessionSettings) {
             String beginString = fixBean.getBeginString();
+            String defaultApplVerID = fixBean.getDefaultApplVerID();
             String dictionary = Objects.requireNonNull(dictionaries.get(beginString), () -> "No dictionary for: " + beginString);
-            fixBean.setDataDictionary(dictionary);
+            if (beginString.equals("FIXT.1.1")) {
+                String appDataDictionary = Objects.requireNonNull(dictionaries.get("FIX.5.0"), () -> "No dictionary for: " + defaultApplVerID);
```

Comment:
Why do we use `FIX.5.0` as a key when getting a dictionary, but then use `defaultApplVerID` in error message?

Reply:
I'm not sure what the dictionaries for  sp1 and sp2 will be called.
should I change the "FIX.5.0" on defaultApplVerID or vice versa

Reply:
If will want to allow using SP1, SP2, etc. dictionaries together then we should add defaultApplVerID to their filename